public/gameInfo.php

```diff
@@ -56,6 +57,56 @@
 
 $gameAlts = getGameAlternatives($gameID);
 
+$v = requestInputSanitized('v', 0, 'integer');
+if ($v != 1) {
+    foreach ($gameAlts as &$gameAlt) {
+        if ($gameAlt['Title'] == '[Theme - Mature]') {
+            if (getAccountDetails($user, $accountDetails) &&
+          BitSet($accountDetails['websitePrefs'], UserPref::SiteMsgOff_MatureContent)) {
+                break;
+            }
+
+            RenderHtmlStart(true); ?>
+<head prefix="og: http://ogp.me/ns# retroachievements: http://ogp.me/ns/apps/retroachievements#">
+    <?php RenderSharedHeader(); ?>
+    <?php RenderTitleTag($pageTitle); ?>
+    <?php RenderGoogleTracking(); ?>
+</head>
+<body>
+<?php RenderTitleBar($user, $points, $truePoints, $unreadMessageCount, $errorCode, $permissions); ?>
+<?php RenderToolbar($user, $permissions);
+            echo "<div id='mainpage'>";
+            echo "<div id='leftcontainer'>";
+            echo "<div class='navpath'>";
+            echo "<a href='/gameList.php'>All Games</a>";
+            echo " &raquo; <a href='/gameList.php?c=$consoleID'>$consoleName</a>";
+            echo " &raquo; <b>$gameTitle</b>";
+            echo "</div>";
+            echo "<h3 class='longheader'>$pageTitle</h3>"; ?>
+  <h4>WARNING: THIS GAME MAY CONTAIN CONTENT NOT APPROPRIATE FOR ALL AGES.</h4>
+  <br/>
+  <div id="confirmation">
+    Are you sure that you want to view this game?
+    <br/>
+    <br/>
+    <form id='consentform' action='/game/<?php echo $gameID ?>' method='get' style='float:left'>
+      <input type='hidden' name='v' value='1'/>
+      <input type='submit' value='Yes. I&apos;m an adult'/>
+    </form>
+    <form id='escapeform' action='/gameList.php' method='get' style='float:left; margin-left:16px'>
+      <input type='hidden' name='c' value='<?php echo $consoleID ?>'/>
+      <input type='submit' value='Not Interested'/>
+    </form>
+  </div>
+</div>
```

Comment:
Missing another `</div>` here which is causing the footer to become distorted.

Reply:
good call. will fix.